packages/account/src/Sections/Assessment/FinancialAssessment/financial-assessment.tsx

```diff
@@ -342,6 +342,14 @@ const FinancialAssessment = observer(() => {
         return '80px';
     };
 
+    const getFormattedOccupationList = values =>
+        values?.employment_status === 'Employed'
+            ? getOccupationList().filter(item => item.value !== 'Unemployed')
+            : getOccupationList();
+
+    const getFormattedOccupationValues = values =>
+        values?.employment_status === 'Employed' && values?.occupation === 'Unemployed' ? '' : values?.occupation;
```

Comment:
I think we need to stop hard-coding the values such as `employed` and `unemployed`. 

There are being hard-coded multiple times in this PR!